test/MonadFix.hs

```diff
@@ -0,0 +1,29 @@
+{-# LANGUAGE RankNTypes #-}
+module MonadFix
+( test
+) where
+
+import Control.Monad (liftM)
```

Comment:
I know this is the same as `fmap`, but I wanted to keep the properties as syntactically similar to how the laws are expressed as possible.